crates/next-core/src/next_client/context.rs

```diff
@@ -281,6 +281,10 @@ pub async fn get_client_module_options_context(
             enable_typeof_window_inlining: Some(TypeofWindow::Object),
             ..Default::default()
         },
+        css: CssOptionsContext {
+            enable_css_transform: true,
+            ..Default::default()
```

Comment:
This should also have use_swc_css, shouldn't it?

Reply:
Slightly further down in the next `let module_options_context` statement, `use_swc_css` is set. Apparently it has so far not been applied to `foreign_codes_options_context`?